samples/rest/files.sh

```diff
@@ -243,6 +243,58 @@ echo
 jq ".candidates[].content.parts[].text" response.json
 # [END files_create_video]
 
+echo "[START files_create_pdf]"
+# [START files_create_pdf]
+MIME_TYPE=$(file -b --mime-type "${PDF_PATH}")
```

Comment:
nit: this is good for audio/image/video which can be lots of different types. But all pdfs are application/pdf. Simplify?